boxitcsv.js

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env node
```

Comment:
do not create a replica of another file. 
option 1: create a function for csv parsing and add the logic to detect csv file in the argument in `main()`
option 2: import `draw...` functions into this file